.vscode/settings.json

```diff
@@ -89,8 +89,8 @@
   "eslint.format.enable": true,
   "eslint.lintTask.enable": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll.markdownlint": true, // first run the markdown linter
-    "editor.action.fixAll": true, // Run the default formatter before eslint, it handles whitespace better
-    "source.fixAll.eslint": true // then run ESLint auto-fix
+    "source.fixAll.markdownlint": "explicit",
```

Comment:
Trivial, but are we going to change the default configurations?

Reply:
VS Code is doing that automatically. No idea how/why.

Reply:
ok.